Backend/MySQL/7.sql

```diff
@@ -0,0 +1 @@
+SELECT DISTINCT USER.NAME FROM `meh`.USER
```

Comment:
Agreed for 11th lab
But generally looks like name should be unique in the system